server/accounts.go

```diff
@@ -3281,3 +3571,185 @@ func (dr *CacheDirAccResolver) Start(s *Server) error {
 	s.Noticef("Managing some jwt in exclusive directory %s", dr.directory)
 	return nil
 }
+
+func (dr *CacheDirAccResolver) Reload() error {
+	return dr.DirAccResolver.Reload()
+}
+
+// Transforms for arbitrarily mapping subjects from one to another for maps, tees and filters.
```

Comment:
What is a tee and filter? (I have an idea but I don't find a test or description how that'd be configured)

is a tee a weighted mapping with 200%?
from one subject to more than one other?
or a weighted mapping with > 100%? (say send all traffic from foo to foo but route one percent to a special subject?)

Reply:
Tees or Filters not impl in this PR. Prefer to keep them there but can remove the comment.

Reply:
was  curios as I read in a comment smth that I did not see in code.
For tee, would have expected a mapping to return more than one string.